judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ProblemItemStatementCard/ProblemItemStatementCard.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Card } from '@blueprintjs/core';
+import { Item } from 'modules/api/sandalphon/problemBundle';
+import { HtmlText } from 'components/HtmlText/HtmlText';
+
+export type ProblemItemStatementCardProps = Item & { className?: string };
+
+export class ProblemItemStatementCard extends React.PureComponent<ProblemItemStatementCardProps> {
+  render() {
+    try {
+      const config: { statement: string } = JSON.parse(this.props.config);
```

Comment:
@fushar, should we convert `config` to JSON in Uriel or in the frontend? At first I thought we should do it in the frontend, but it turns out that Uriel needs to parse the config anyway for grading the submissions.

@jauhararifin if we're going to convert `config` on the frontend, perhaps the `JSON.parse()` can be done in the function that calls the API?

Reply:
I think for now it's OK to parse in FE. And yes, this should be done in the function that calls the API. So that this component will receive the config that is already cast to the correct type.

Reply:
So I just leave it this way now?